functional-tests/entry.py

```diff
@@ -45,7 +46,7 @@ def gen_ext_btc_address(self) -> str | List[str]:
         self._ext_btc_addr_idx += 1
         return tr_addr
 
-    def gen_rec_btc_address(self) -> str | List[str]:
+    def gen_rec_btc_address(self) -> str | list[str]:
```

Comment:
If I am not mistaken these `ruff` fixes were already done in #511. You should rebase.

Reply:
It's possible some were missed.